libnavigation-base/src/test/java/com/mapbox/navigation/base/options/PredictiveCacheOptionsBuilderTest.kt

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.navigation.base.options
+
+import com.mapbox.navigation.testing.BuilderTest
+import org.junit.Test
+import kotlin.reflect.KClass
+
+class PredictiveCacheOptionsBuilderTest :
+    BuilderTest<PredictiveCacheOptions, PredictiveCacheOptions.Builder>() {
+    override fun getImplementationClass(): KClass<PredictiveCacheOptions> =
+        PredictiveCacheOptions::class
+
+    override fun getFilledUpBuilder(): PredictiveCacheOptions.Builder =
+        PredictiveCacheOptions.Builder().apply {
+            predictiveCacheNavigationOptions(
+                PredictiveCacheNavigationOptions.Builder().apply {
+                    predictiveCacheLocationOptions(
+                        PredictiveCacheLocationOptions.Builder().apply {
+                            currentLocationRadiusInMeters(300)
+                            routeBufferRadiusInMeters(50)
+                            destinationLocationRadiusInMeters(20)
+                        }.build()
+                    )
+                }.build()
+            )
+            predictiveCacheMapsOptions(
```

Comment:
should we also test `predictiveCacheMapsOptionsList` here?

Reply:
Thanks, it actually found an error.